src/AutosizeInput.js

```diff
@@ -96,10 +96,12 @@ const AutosizeInput = React.createClass({
 		const inputStyle = Object.assign({}, this.props.inputStyle);
 		inputStyle.width = this.state.inputWidth + 'px';
 		inputStyle.boxSizing = 'content-box';
+		const { defaultValue, onChange, value } = this.props;
+		const inputProps = { defaultValue, onChange, value, placeholder: this.props.placeholder };
```

Comment:
The general change makes sense, but why would you not either create a `placeholder` variable const the same as `defaultValue` etc above, or create each property in the `inputProps` object from `this.props.x` as per `placeholder`? The inconsistency here is throwing me off. Is there something I'm missing?